tests/python_tests/test_continuous_batching.py

```diff
@@ -417,6 +405,7 @@ def py_streamer(py_str: str):
         return False
 
     _ = pipe.generate(input, generation_config=generation_config, streamer=py_streamer)
+    _ = pipe.generate(input, generation_config=generation_config, streamer=py_streamer)
```

Comment:
duplicate

please, review PR by yourself as well, because is some other places we also have duplicated lines

Reply:
merge tool resolves the conflicts in such way. Happens